src/Lecture3.hs

```diff
@@ -92,7 +101,14 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+-- daysTo :: (Bounded a, Enum a) => a -> a -> Int
+daysTo :: Weekday -> Weekday -> Int
+daysTo from to = delta
+  where from' = fromEnum from
+        to' = fromEnum to
+        final = fromEnum (maxBound :: Weekday)
+        delta = if to' >= from' then (to' - from')
+                else final - (from' - to') + 1
```

Comment:
That's a correct implementation! You can simplify it even further if you remember how `mod` works for negative numbers 🙂